### DISCUSSION TITLE
How to compute soil moisture for 60 cm depth.

### DISCUSSION THREAD

**Discussion by Anujiitgn** [iiasa/CWatM#29] [Q&A]
Hi everyone,

Hope you all are doing well.
I extracted data from output variables; 

Sum_w1, sum_w2 and sum_w3

But I want to compute soil moisture for specific depth in my case it is 60 cm.

Please help me this regard.

Thankyou
Anuj Prakash

**Comment by guillaumot** [comment]
Hi,
To compute soil moisture (as water volume / soil volume), you will need the variables rootDepth[0], rootDepth[1], rootDepth[2], they äre the thickness of each soil layer. And w1, w2 and w3 instead of sum_w1, sum_w2 and sum_w3

Assuming that 60cm depth is in the second soil layer (index 1) because  rootDepth[0] > 0.6 > rootDepth[0] + rootDepth[1].
You can obtain the soil moisture within the second soil layer for each landcover type by:
 - forest:  w2[0] / rootDepth[1][0]
 - grasslands:  w2[1] / rootDepth[1][1]
 - paddy irrigation crops: w2[2] / rootDepth[1][2]
 - non-paddy irrigation crops: w2[3] /rootDepth[1][3].

Thus if you need soil moisture in the second layer in grasslands you will need to extract: w2[1] and rootDepth[1][1]

I would suggest to not use sum_w because it integrates different landcovers.

So, you can not obtain soil moisture at a specific depth, you can only get one average value inside a soil layer (between 0.05 and 1 m for example)

Let me know if it works!

Best,
Luca
